msal/src/MSAL.Common.props

```diff
@@ -182,6 +180,7 @@
     <PackageReference Include="System.Runtime.Serialization.Json" Version="4.3.0" />
     <PackageReference Include="System.Xml.XDocument" Version="4.3.0" />
     <PackageReference Include="System.Net.NameResolution" Version="4.3.0" />
+    <PackageReference Include="System.Security.SecureString" Version="4.3.0" />
```

Comment:
Unfortunately we have to take a dependency on SecureString in order to throw nice exceptions on Android, UWP and iOS...